editor/src/core/workers/parser-printer/parser-printer.ts

```diff
@@ -564,19 +572,10 @@ function printStatements(statements: Array<TS.Node>, shouldPrettify: boolean): s
     false,
     TS.ScriptKind.TS,
   )
-  const printFlags =
-    TS.ListFormat.Indented |
-    TS.ListFormat.MultiLine |
-    TS.ListFormat.SpaceBetweenBraces |
-    TS.ListFormat.MultiLineBlockStatements |
-    TS.ListFormat.PreferNewLine |
-    TS.ListFormat.MultiLineFunctionBodyStatements |
-    TS.ListFormat.MultiLineTypeLiteralMembers
-  const typescriptPrintedResult = printer.printList(
-    printFlags,
-    TS.createNodeArray(statements),
-    resultFile,
-  )
+  const printedParts = statements.map((statement) => {
+    return printer.printNode(TS.EmitHint.Unspecified, statement, resultFile)
```

Comment:
What's the deal with removing all of the flags here?

Reply:
See last commit detail bullet point.